### PR TITLE
Dhruv/Account-banner-modification/TRAH-4030

### PR DESCRIPTION
**Pull request by Dhruv-deriv**
Implemented the Status banner with the updated design 

Screenshots: 
1.
Screenshot 2024-09-02 at 2 10 07 PM

2.
Screenshot 2024-09-02 at 2 10 50 PM

3.
Screenshot 2024-09-02 at 2 11 47 PM

### CODE REVIEW

File: packages/account/src/Configs/get-mt5-status-badge-config.tsx [hunk 1]
```diff
@@ -12,17 +12,25 @@ const getMT5StatusBadgeConfig = (mt5_account_status: TMT5AccountStatus) => {
             return {
                 text: (
                     <Localize
-                        i18n_default_text='<0>Pending verification</0>'
-                        components={[<Text key={0} weight='bold' size='xxxs' color='var(--status-warning)' />]}
+                        i18n_default_text='<1>In review</1>'
+                        components={[
+                            <Text
+                                key={0}
```
Comment (amina-deriv) [inline packages/account/src/Configs/get-mt5-status-badge-config.tsx]: @Dhruv-deriv do we need this Text component? There is no key with key 0

File: packages/account/src/Configs/get-mt5-status-badge-config.tsx [hunk 2]
```diff
@@ -12,17 +12,25 @@ const getMT5StatusBadgeConfig = (mt5_account_status: TMT5AccountStatus) => {
...
+                icon: 'IcMt5Pending',
             };
         case MT5_ACCOUNT_STATUS.FAILED:
             return {
                 text: (
                     <Localize
-                        i18n_default_text='<0>Verification failed.</0> <1>Why?</1>'
+                        i18n_default_text='<1>Failed</1>'
                         components={[
                             <Text key={0} weight='bold' size='xxxs' color='var(--status-danger)' />,
```
Comment (amina-deriv) [inline packages/account/src/Configs/get-mt5-status-badge-config.tsx]: same here

File: packages/account/src/Configs/get-mt5-status-badge-config.tsx [hunk 3]
```diff
@@ -35,13 +43,13 @@ const getMT5StatusBadgeConfig = (mt5_account_status: TMT5AccountStatus) => {
...
+                icon: 'IcMt5Failed',
             };
         case MT5_ACCOUNT_STATUS.NEEDS_VERIFICATION: {
             return {
                 text: (
                     <Localize
-                        i18n_default_text='<0>Needs verification.</0><1>Verify now</1>'
+                        i18n_default_text='<1>Needs Verification</1>'
                         components={[
                             <Text key={0} weight='bold' size='xxxs' color='var(--status-info)' />,
```
Comment (amina-deriv) [inline packages/account/src/Configs/get-mt5-status-badge-config.tsx]: same

File: packages/account/src/Configs/get-mt5-status-badge-config.tsx (hunk 1 above)
Comment (yauheni-deriv) [reply packages/account/src/Configs/get-mt5-status-badge-config.tsx]: can we use here Localize inside text component? <Text><Localize/></Text>?

File: packages/components/src/components/status-badge/status-badge.scss [hunk 4]
```diff
@@ -12,9 +12,23 @@
         align-items: center;
         white-space: nowrap;
 
+        &--pending {
+            font-size: var(--text-size-xxxs);
+            color: var(--status-warning);
+            background-color: var(--status-warning-transparent);
+            border: 1.5px;
```
Comment (yauheni-deriv) [inline packages/components/src/components/status-badge/status-badge.scss]: is it really expected?
I think all our deriv CSS values should be multiple of 8, so we may use less than 8, but it should be 4 or 2

File: packages/account/src/Configs/get-mt5-status-badge-config.tsx (hunk 3 above)
Comment (Dhruv-deriv) [reply packages/account/src/Configs/get-mt5-status-badge-config.tsx]: done 👍

File: packages/account/src/Configs/get-mt5-status-badge-config.tsx (hunk 2 above)
Comment (Dhruv-deriv) [reply packages/account/src/Configs/get-mt5-status-badge-config.tsx]: done 👍

File: packages/account/src/Configs/get-mt5-status-badge-config.tsx (hunk 1 above)
Comment (Dhruv-deriv) [reply packages/account/src/Configs/get-mt5-status-badge-config.tsx]: done 👍

File: packages/components/src/components/status-badge/status-badge.scss (hunk 4 above)
Comment (Dhruv-deriv) [reply packages/components/src/components/status-badge/status-badge.scss]: hey @yauheni-deriv is there any particular reason why we should stick to multiples of 8 or values less than 2

File: packages/account/src/Configs/get-mt5-status-badge-config.tsx [hunk 5]
```diff
@@ -10,72 +10,54 @@ const getMT5StatusBadgeConfig = (mt5_account_status: TMT5AccountStatus) => {
...
             return {
                 text: <Localize i18n_default_text='<0>Account closed</0>' components={[BadgeTextComponent]} />,
                 icon: 'IcAlertWarning',
+                icon_size: '19',
             };
         case MT5_ACCOUNT_STATUS.UNDER_MAINTENANCE:
             return {
                 text: <Localize i18n_default_text='<0>Server maintenance</0>' components={[BadgeTextComponent]} />,
                 icon: 'IcAlertWarning',
+                icon_size: '19',
```
Comment (amina-deriv) [inline packages/account/src/Configs/get-mt5-status-badge-config.tsx]: Screenshot 2024-09-09 at 10 00 37 AM
can you double check the icon sizes? as per design it should be 12 and 18

File: packages/components/src/components/status-badge/status-badge.scss (hunk 4 above)
Comment (amina-deriv) [reply packages/components/src/components/status-badge/status-badge.scss]: As per design, it is 4 @Dhruv-deriv . could you please check

Comment (yauheni-deriv) [reply packages/components/src/components/status-badge/status-badge.scss]: @Dhruv-deriv , this is kind of logic and steps, which our designers follow

File: packages/account/src/Configs/get-mt5-status-badge-config.tsx [hunk 6]
```diff
@@ -10,72 +10,54 @@ const getMT5StatusBadgeConfig = (mt5_account_status: TMT5AccountStatus) => {
...
             };
         case TRADING_PLATFORM_STATUS.UNAVAILABLE:
             return {
                 text: <Localize i18n_default_text='<0>Unavailable</0>' components={[BadgeTextComponent]} />,
                 icon: 'IcAlertWarning',
+                icon_size: '2',
```
Comment (amina-deriv) [inline packages/account/src/Configs/get-mt5-status-badge-config.tsx]: remove this

File: packages/account/src/Configs/get-mt5-status-badge-config.tsx [hunk 7]
```diff
@@ -10,72 +10,54 @@ const getMT5StatusBadgeConfig = (mt5_account_status: TMT5AccountStatus) => {
...
             return {
                 text: <Localize i18n_default_text='<0>No new positions</0>' components={[BadgeTextComponent]} />,
                 icon: 'IcAlertWarning',
+                icon_size: '19',
             };
         case MT5_ACCOUNT_STATUS.MIGRATED_WITHOUT_POSITION:
             return {
                 text: <Localize i18n_default_text='<0>Account closed</0>' components={[BadgeTextComponent]} />,
                 icon: 'IcAlertWarning',
+                icon_size: '19',
```
Comment (amina-deriv) [inline packages/account/src/Configs/get-mt5-status-badge-config.tsx]: remove this

Comment (Dhruv-deriv) [reply packages/account/src/Configs/get-mt5-status-badge-config.tsx]: done

File: packages/account/src/Configs/get-mt5-status-badge-config.tsx (hunk 6 above)
Comment (Dhruv-deriv) [reply packages/account/src/Configs/get-mt5-status-badge-config.tsx]: done

File: packages/components/src/components/status-badge/status-badge.scss (hunk 4 above)
Comment (Dhruv-deriv) [reply packages/components/src/components/status-badge/status-badge.scss]: its fixed

File: packages/account/src/Configs/get-mt5-status-badge-config.tsx (hunk 5 above)
Comment (Dhruv-deriv) [reply packages/account/src/Configs/get-mt5-status-badge-config.tsx]: Screenshot 2024-09-09 at 4 32 06 PM

yes the icon is of that size

Comment (Dhruv-deriv) [reply packages/account/src/Configs/get-mt5-status-badge-config.tsx]: i have made it 18

**Review by amina-deriv — APPROVED** [review]

**Pull request merged**